app/javascript/controllers/creator_settings_controller.js

```diff
@@ -0,0 +1,62 @@
+import { Controller } from '@hotwired/stimulus';
```

Comment:
@nickytonline will we move this controller to `/admin` in this PR as we discussed yesterday or in a subsequent one?

Reply:
Oh yeah. Thanks for the reminder. Will do.

Reply:
Done.